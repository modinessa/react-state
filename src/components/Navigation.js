import React from "react";
import { Link } from "react-router-dom";

export function Navigation() {
	return (
		<div className="app-navigation">
			<Link to="/" className="app-navigation__logo" href="#">
				Project
			</Link>

			<nav className="app-navigation__nav">
				<ul className="app-navigation__nav-list">
					<Link to="community" className="app-navigation__nav-list-item">
						Community
					</Link>

					<Link to="not-found" className="app-navigation__nav-list-item">
						About us
					</Link>

					<Link to="not-found" className="app-navigation__nav-list-item">
						Projects
					</Link>

					<Link to="not-found" className="app-navigation__nav-list-item">
						Contact
					</Link>
				</ul>
			</nav>

			<div className="app-navigation__nav-menu-button">
				<div className="app-navigation__nav-menu-button-icon">
					<span></span>
					<span></span>
					<span></span>
				</div>
			</div>
		</div>
	)
}
