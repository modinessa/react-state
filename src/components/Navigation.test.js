import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom"; 

import {Navigation} from './Navigation';

describe('Navigation', () => {
	it('renders component', () => {
		expect(renderer.create(
			<BrowserRouter>
				<Navigation/>
			</BrowserRouter>)).toMatchSnapshot();
	});
});
