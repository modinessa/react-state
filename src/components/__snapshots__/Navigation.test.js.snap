// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders component 1`] = `
<div
  className="app-navigation"
>
  <a
    className="app-navigation__logo"
    href="/"
    onClick={[Function]}
  >
    Project
  </a>
  <nav
    className="app-navigation__nav"
  >
    <ul
      className="app-navigation__nav-list"
    >
      <a
        className="app-navigation__nav-list-item"
        href="/community"
        onClick={[Function]}
      >
        Community
      </a>
      <a
        className="app-navigation__nav-list-item"
        href="/about us"
        onClick={[Function]}
      >
        About us
      </a>
      <a
        className="app-navigation__nav-list-item"
        href="/projects"
        onClick={[Function]}
      >
        Projects
      </a>
      <a
        className="app-navigation__nav-list-item"
        href="/contact"
        onClick={[Function]}
      >
        Contact
      </a>
    </ul>
  </nav>
</div>
`;
